test/test_nn.py

```diff
@@ -3293,7 +3293,7 @@ def _create_src_lengths_mask(batch_size, src_lengths):
             return (src_indices < src_lengths).int().detach()
 
         def _multihead_attn_test_helper(add_key_padding_mask=False, add_bias_kv=False, add_zero_attn=False,
-                                        saved_kv=False, same_embed_dim=False):
+                                        saved_kv=False, same_embed_dim=False, byte_mask=False):
```

Comment:
Is this byte_mask argument actually necessary? why not dispatch on the type of the mask